storage/memory/storage.go

```diff
@@ -95,7 +95,7 @@ func (o *ObjectStorage) Set(obj core.Object) (core.Hash, error) {
 }
 
 // Get returns a object with the given hash
-func (o *ObjectStorage) Get(h core.Hash) (core.Object, error) {
+func (o *ObjectStorage) Get(h core.Hash, t core.ObjectType) (core.Object, error) {
```

Comment:
Already documented in core/storage.go, which is the interface. I don't think I have to duplicate it in every implementation, right?